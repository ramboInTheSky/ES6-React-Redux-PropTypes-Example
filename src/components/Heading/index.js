import styled from 'styled-components';
import { Typography } from 'nhh-styles';

export default styled(Typography.H3)`
  margin-bottom: 1.5rem;
`;
