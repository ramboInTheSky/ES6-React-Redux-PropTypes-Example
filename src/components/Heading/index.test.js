import React from 'react';
import { shallow } from 'enzyme';

import 'jest-styled-components';

import Heading from './';

describe('Heading', () => {
  it('should render correctly', () => {
    expect(shallow(<Heading />)).toMatchSnapshot();
  });
});
