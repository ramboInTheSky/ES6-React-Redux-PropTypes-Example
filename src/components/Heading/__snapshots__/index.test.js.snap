// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render correctly 1`] = `
.c0 {
  margin-bottom: 1.5rem;
}

<styled.h3
  className="c0"
/>
`;
